
From: Andrey Panin <pazke@donpac.ru>

BROKEN_ACPI_Sx flag doesn't seem to be used anywhere in the kernel, so ASUS
K7V-RM can be removed.

Signed-off-by: Andrey Panin <pazke@donpac.ru>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/i386/kernel/dmi_scan.c |   17 -----------------
 1 files changed, 17 deletions(-)

diff -puN arch/i386/kernel/dmi_scan.c~267-mm1-remove-unused-asus-k7v-rm-dmi-quirk arch/i386/kernel/dmi_scan.c
--- 25/arch/i386/kernel/dmi_scan.c~267-mm1-remove-unused-asus-k7v-rm-dmi-quirk	Wed Jun 23 14:18:27 2004
+++ 25-akpm/arch/i386/kernel/dmi_scan.c	Wed Jun 23 14:18:27 2004
@@ -348,17 +348,6 @@ static __init int swab_apm_power_in_minu
 }
 
 /*
- * ASUS K7V-RM has broken ACPI table defining sleep modes
- */
-
-static __init int broken_acpi_Sx(struct dmi_blacklist *d)
-{
-	printk(KERN_WARNING "Detected ASUS mainboard with broken ACPI sleep table\n");
-	dmi_broken |= BROKEN_ACPI_Sx;
-	return 0;
-}
-
-/*
  * Toshiba keyboard likes to repeat keys when they are not repeated.
  */
 
@@ -715,12 +704,6 @@ static __initdata struct dmi_blacklist d
 			NO_MATCH, NO_MATCH
 			} },
 
-	{ broken_acpi_Sx, "ASUS K7V-RM", {		/* Bad ACPI Sx table */
-			MATCH(DMI_BIOS_VERSION,"ASUS K7V-RM ACPI BIOS Revision 1003A"),
-			MATCH(DMI_BOARD_NAME, "<K7V-RM>"),
-			NO_MATCH, NO_MATCH
-			} },
-
 	{ broken_toshiba_keyboard, "Toshiba Satellite 4030cdt", { /* Keyboard generates spurious repeats */
 			MATCH(DMI_PRODUCT_NAME, "S4030CDT/4.3"),
 			NO_MATCH, NO_MATCH, NO_MATCH
_
